res/database/Default/database_ads.xml

```diff
@@ -1543,7 +1545,7 @@ Available in any well-stocked drugstore.</en>
 			</title>
 			<description>
 				<de>Das Handbuch zum Umrechnen von Mark der DDR in D-Mark! Für alle, die's genau wissen wollen. Weil früher alles besser war. Und billiger?</de>
-				<en>The manual for converting East German mark (DDM) to Deutsche Mark (DEM)! For those who want to know exactly. Because everything used to be better. And cheaper?</en>
+				<en>The manual for converting East German mark (DDM) into Deutsche Mark (DEM)! For all who want to know things precisely. Because everything was better in the old days. And cheaper?</en>
```

Comment:
For consistency with EUR, it should be the other way around. Convertinv DEM int DDM (in German as well)

Reply:
fine by me. Where's Sjaele? `: )`

Reply:
Think he is still trying to do music and enjoying his life - without TVTower.